RHTAPINST-149: Wait for Operators Rollout

otaviof opened the pull request:
All OLM managed operator have a "olm.managed=true" label, so we can use this to wait for the rollout of the operator to complete.

Roming22 commented on installer/config.yaml:
You might want to revert those changes

otaviof replied:
Indeed, please consider latest changes.

otaviof committed:
feat: Wait for Operators Rollout

All OLM managed operator have a "olm.managed=true" label, so we can use
this to wait for the rollout of the operator to complete.

otaviof committed:
docs: `rhtap-subscriptions` README.md

otaviof commented:
/retest

Roming22 commented:
/lgtm

Roming22 commented:
/retest

Roming22 commented:
/lgtm
